### DISCUSSION TITLE
Purchased a team plan

### DISCUSSION THREAD

**Discussion by kwlee-apm** [Budibase/budibase#9484] [Help]
I configured it as self-host.
And I purchased the paid service Team plan. 

Where can I get the activation code?

**Comment by shogunpurple** [comment]
Hi @kwonwoo0078 - you can find your license key at https://account.budibase.app/portal/upgrade

**Reply by dannyshin76** [reply]
On that screen, could you point me where I can click to find the license key for local hosting? All I can see is just:
![image](https://user-images.githubusercontent.com/93232582/215933126-b5e293fe-f60d-4b8c-8435-4ec2161171ec.png)
